packages/vital-navigation/src/components/Breadcrumbs/tokens/vitalBreadcrumbs.ts

```diff
@@ -63,9 +61,7 @@ const Base = asBreadcrumbsToken({
   Theme: {
     StartingTrail: vitalColor.TextPrimaryInteractiveHover,
     Title: vitalColor.TextPrimaryInteractiveHover,
-    FinalTrail: withDesign({
-      Title: vitalTextDecoration.Bold,
-    }),
+    Item: addClasses('last:font-bold'),
```

Comment:
remove addClasses() and it can just take string

Reply:
good catch, updated, thanks